doc/esmvalcore/preprocessor.rst

```diff
@@ -138,344 +260,559 @@ losing generality. To mask ice out, `mask_landseaice` can be used:
-| `4. extract_named_regions`_: extract a region from a cube given its name.
 
-1. extract_region
------------------
+``extract_region``
+------------------
 
 This function masks data outside a rectagular region requested. The boundairies
 of the region are provided as latitude and longitude coordinates in the
 arguments:
 
-* start_longitude
-* end_longitude
-* start_latitude
-* end_latitude
+* ``start_longitude``
+* ``end_longitude``
+* ``start_latitude``
+* ``end_latitude``
 
 Note that this function can only be used to extract a rectangular region.
 
 See also :func:`esmvalcore.preprocessor.extract_region`.
 
-2. zonal_means
---------------
+
+``zonal_means``
+---------------
 
 The function calculates the zonal or meridional means. While this function is
-named `zonal_mean`, it can be used to apply several different operations in
-an zonal or meridional direction.
-This function takes two arguments:
+named ``zonal_mean``, it can be used to apply several different operations in
+an zonal or meridional direction. This function takes two arguments:
 
-* coordinate: Which direction to apply the operation: latitude or longitude.
-* mean_type: Which operation to apply: mean, std_dev, variance, median, min or
-* max. 
+* ``coordinate``: Which direction to apply the operation: latitude or longitude
+* ``mean_type``: Which operation to apply: mean, std_dev, variance, median, min
+  or max
 
 See also :func:`esmvalcore.preprocessor.zonal_means`.
 
-3. area_statistics
-------------------
 
-This function calculates the average value over a region - weighted by the
-cell areas of the region.
+``area_statistics``
+-------------------
 
-This function takes one argument:
-* operator: the name of the operation to apply.
+This function calculates the average value over a region - weighted by the cell
+areas of the region. This function takes the argument, ``operator``: the name
+of the operation to apply.
 
 This function can be used to apply several different operations in the
-horizonal plane: mean, standard deviation, median variance, minimum and
-maximum.
+horizonal plane: mean, standard deviation, median variance, minimum and maximum.
 
 Note that this function is applied over the entire dataset. If only a specific
 region, depth layer or time period is required, then those regions need to be
 removed using other preprocessor operations in advance.
 
 See also :func:`esmvalcore.preprocessor.area_statistics`.
 
-4. extract_named_regions
-------------------------
-
-This function extract a specific named region from the data. 
-This function takes onw argument: 
 
-* regions: either a string or a list of strings of named regions. 
+``extract_named_regions``
+-------------------------
 
-Note that the dataset must have a `region` cooordinate which includes a list of
-strings as values. This function then matches the named regions against the
-requested string.
+This function extract a specific named region from the data. This function
```

Comment:
Should be "... function extracts..." instead of "... function extracts..."?

Reply:
:+1: